src/components/Tooltip/ControlledTooltip.js

```diff
@@ -173,6 +179,10 @@ ControlledTooltip.propTypes = {
    * Controls whether the popover is visible
    */
   isOpen: PropTypes.bool,
+  /**
+   * Allows you to set popper offsets manually
```

Comment:
Please mention the string format here: _'n, n'_

Reply:
Added documentation link bellow